### ISSUE TITLE
Unable to create Expo Config Plugin using expo-module-scripts

### ISSUE THREAD

**Issue by gerzonc** [expo/expo#19066]
### Expected behavior

Create Expo Config Plugin basic configuration by running `yarn`

### Actual behavior

When running `yarn`, I get this output on my project

```bash
> expo-module prepare

Configuring module
Tried to guess the docs homepage for react-native-razorpay; add it under the "homepage" entry in package.json
expo-module-scripts: README.md exists, not updating
error TS18003: No inputs were found in config file '/Users/MY_USER/Desktop/LIBRARY_NAME/tsconfig.json'. Specified 'include' paths were '["./src"]' and 'exclude' paths were '["**/__mocks__/*","**/__tests__/*","**/__stories__/*"]'.
```

### What platform(s) does this occur on?

Not Applicable

### Environment

 System:
      OS: macOS 12.5.1
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.6.1 - ~/.nvm/versions/node/v16.6.1/bin/node
      Yarn: 1.22.10 - /usr/local/bin/yarn
      npm: 7.20.3 - ~/.nvm/versions/node/v16.6.1/bin/npm
      Watchman: 2022.07.04.00 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.11.2 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    IDEs:
      Android Studio: 2021.2 AI-212.5712.43.2112.8609683
      Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
    npmGlobalPackages:
      eas-cli: 0.60.0
      expo-cli: 6.0.2
    Expo Workflow: managed

### Minimal reproducible example

I'm not even sure if this applies to this or not, here is a Snack: https://snack.expo.dev/@gerzonc/84e33d

Basically issue is that once you run `yarn` or `expo-module prepare` it adds `tsconfig.json`, a `README.md`, an `.eslintrc.js` and after that it drops the error I mentioned above

**Comment by byCedric** [comment]
Hi @gerzonc, I'm not sure if I follow. Let me try to summarize it:
- Trying to create an Expo config plugin using `expo-module-scripts`
- When running `yarn expo-module prepare`, it throws an error
- Is that error gone after re-running `yarn expo-module prepare`, as it adds the missing files?

I'm not sure how to help here without knowing the exact repo structure. The Snack you linked only contains some plugin code, without any traces of `expo-module-scripts`.

**Comment by gerzonc** [comment]
> Hi @gerzonc, I'm not sure if I follow. Let me try to summarize it:
> 
> * Trying to create an Expo config plugin using `expo-module-scripts`
> * When running `yarn expo-module prepare`, it throws an error
> * Is that error gone after re-running `yarn expo-module prepare`, as it adds the missing files?
> 
> I'm not sure how to help here without knowing the exact repo structure. The Snack you linked only contains some plugin code, without any traces of `expo-module-scripts`.

Got it working now, had the wrong file naming structure for the plugin (was using `index.ts` instead of `withMyPlugin.ts`)
